# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.h
# Commit: 810b73719871
# Full Hash: 810b7371987139844429d0206f9da6a7701a1efc
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-23 04:41:59
# Regressor Bug: 1525720
# File Overlap Count: 1
# Description:
#   Bug 1525720, part 13 - Stop inheriting nsIRemoteTab interface in BrowserParent. r=nika
#   
#   This commit removes nsIRemoteTab as a parent class from BrowserParent,
#   so that BrowserHost is the only concrete implementation of nsIRemoteTab.
#   
# ==============================================================================

diff -r ee10a8254481 -r 810b73719871 dom/ipc/ContentParent.h
--- a/dom/ipc/ContentParent.h	Wed May 08 11:45:23 2019 -0500
+++ b/dom/ipc/ContentParent.h	Wed May 08 14:34:47 2019 -0500
@@ -681,7 +681,7 @@
       const bool& aSizeSpecified, nsIURI* aURIToLoad,
       const nsCString& aFeatures, const float& aFullZoom,
       uint64_t aNextRemoteTabId, const nsString& aName, nsresult& aResult,
-      nsCOMPtr<nsIRemoteTab>& aNewBrowserParent, bool* aWindowIsNew,
+      nsCOMPtr<nsIRemoteTab>& aNewRemoteTab, bool* aWindowIsNew,
       int32_t& aOpenLocation, nsIPrincipal* aTriggeringPrincipal,
       nsIReferrerInfo* aReferrerInfo, bool aLoadUri,
       nsIContentSecurityPolicy* aCsp);